pythonforandroid/archs.py

```diff
@@ -82,8 +101,19 @@ def get_env(self, with_flags_in_cc=True):
             env['NDK_CCACHE'] = self.ctx.ccache
             env.update({k: v for k, v in environ.items() if k.startswith('CCACHE_')})
 
-        cc = find_executable('{command_prefix}-gcc'.format(
-            command_prefix=command_prefix), path=environ['PATH'])
+        if clang:
+            clang_path = join(
+                self.ctx.ndk_dir, 'toolchains', 'llvm', 'prebuilt',
```

Comment:
FYI I had to symlink `llvm-3.7` to `llvm`:
```
ln -sfn ~/.buildozer/crystax-ndk-10.3.2/toolchains/llvm-3.7/ ~/.buildozer/crystax-ndk-10.3.2/toolchains/llvm
```
We may not want to tackle it in this PR, it's just for us to know

Reply:
mmm...you are right, I did not consider crystax and I see that the paths are different, do you want that I fix it?

Something like:
```
llvm_dirname = 'llvm'
if self.ctx.python_recipe and self.ctx.python_recipe.from_crystax:
    llvm_dirname = split(glob(join(self.ctx.ndk_dir, 'toolchains', 'llvm-*'))[-1])[-1]
    clang_path = join(self.ctx.ndk_dir, 'toolchains', llvm_dirname, 'prebuilt', 'linux-x86_64', 'bin')
```

with the proper imports, will solve the problem.

Reply:
Thanks for the reply.
I don't consider it critical since we're dropping crystax support slowly.
But yes still if having it working during the transition is not that painful, then yes.
I like the approach you suggest, however we increase the cyclomatic complexity slightly with that if.
What about going directly for `'llvm*'` (note that I removed the dash `-`) and kill the if? Do you think that could introduce issues to non-crystax NDK or you it should be safe?
So basically the code would be:
```python
llvm_dirname = split(glob(join(self.ctx.ndk_dir, 'toolchains', 'llvm*'))[-1])[-1]
clang_path = join(self.ctx.ndk_dir, 'toolchains', llvm_dirname, 'prebuilt', 'linux-x86_64', 'bin')
```

Reply:
It works, I tested via python console and hardcoded paths. Your solution is far better :smile: thanks!!!